backend/wsgi_handler.py

```diff
@@ -14,9 +14,6 @@
 import traceback
```

Comment:
you can revert those changes

Reply:
I reverted this, I don't know github is still showing these changes...

Reply:
I added another safeguard to the input. I think for the restricted status, I should  create another Type... Will change later maybe